interfaces/builtin/bluez.go

```diff
@@ -225,3 +225,13 @@ func (iface *BluezInterface) AutoConnect(*interfaces.Plug, *interfaces.Slot) boo
 	// allow what declarations allowed
 	return true
 }
+
+func (iface *BluezInterface) ValidatePlug(plug *interfaces.Plug, attrs map[string]interface{}) error {
```

Comment:
I kind of wish we did the same trick we have elsewhere with interfaces. We could have a function that validates a plug that checks if the interface supports validation. This would cut the amount of boilerplate code *significantly* as it could do simple stuff like checking if interfaces match etc etc (sanitize could be done the same way). What do you think?